donkeycar/config.py

```diff
@@ -53,6 +53,8 @@ def load_config(config_path=None, myconfig="myconfig.py"):
             local_config = os.path.join(os.path.curdir, 'config.py')
             if os.path.exists(local_config):
                 config_path = local_config
+    else:
+        config_path = os.path.join(config_path, 'config.py')
```

Comment:
This doesn't look correct, as the variable `config_path` is meant to contain the full path of the `config.py` file. Either you pass this to the function, or it gets auto-generated for you.